mono/mini/mini.c

```diff
@@ -2079,12 +2079,6 @@ mono_compile_create_vars (MonoCompile *cfg)
 		mono_apply_volatile (cfg->locals [i], header->volatile_locals, i);
 	}
 
-	mono_bitset_free (header->volatile_locals);
```

Comment:
I was going to grow the header to hold the bits, and copy them here, had even written that in the initial work, but this should also work.

Reply:
growing for this usecase is not necessary, just makes it more complicated than it needs to be.